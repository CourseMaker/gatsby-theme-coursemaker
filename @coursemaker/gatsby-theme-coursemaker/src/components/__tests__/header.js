import React from 'react';
import renderer from 'react-test-renderer';
import Header from '../header';
import schoolDouble from '../__mocks__/school-double';

describe('Header', () => {
    it('renders correctly', () => {
        // Created using the query from Header.js
        const schoolTestSample = schoolDouble;
        const tree = renderer.create(<Header school={schoolTestSample} />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
