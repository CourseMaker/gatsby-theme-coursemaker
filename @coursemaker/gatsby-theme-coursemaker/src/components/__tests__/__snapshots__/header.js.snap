// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<header
  className="sticky top-0 z-20 shadow-md text-white bg-blue-500"
>
  <div
    className="container mx-auto"
  >
    <div
      className="flex flex-wrap items-center"
    >
      <a
        aria-current="page"
        className="inline-flex items-center h-20 text-lg md:h-24 logo"
        href="/"
        onClick={[Function]}
        onMouseEnter={[Function]}
        style={Object {}}
      >
        <span
          className="font-semibold"
        >
          dsacademy2
        </span>
      </a>
      <nav
        className="items-center order-3 w-full ml-auto list-none lg:flex lg:space-y-0 space-y-3 
						lg:space-x-6 lg:order-1 lg:w-auto py-8 lg:py-0 border-t lg:border-t-0 border-white border-opacity-25 
						hidden"
      >
        <li>
          <a
            className=""
            href="/#overview"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            Overview
          </a>
        </li>
        <li>
          <a
            className=""
            href="/#courses"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            Courses
          </a>
        </li>
        <li>
          <a
            className=""
            href="/login"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            Login
          </a>
        </li>
        <li>
          <a
            className="btn btn-white"
            href="/register"
            onClick={[Function]}
            onMouseEnter={[Function]}
          >
            Enroll Now
          </a>
        </li>
      </nav>
      <button
        className="block w-8 h-6 ml-auto cursor-pointer lg:hidden burger-menu 
						"
        onClick={[Function]}
        tabIndex={0}
      >
        <div
          className="bar"
        />
        <div
          className="bar"
        />
        <div
          className="bar"
        />
      </button>
    </div>
  </div>
</header>
`;
